src/Unused/CLI/Views/SearchResult/ListResult.hs

```diff
@@ -0,0 +1,81 @@
+module Unused.CLI.Views.SearchResult.ListResult
+    ( printList
+    ) where
+
+import Control.Monad (forM_, void)
+import Data.List (intercalate, (\\))
+import Unused.CLI.Util
+import Unused.Types
+import Unused.CLI.Views.SearchResult.Internal
+import Unused.CLI.Views.SearchResult.Types
+
+printList :: TermResults -> [TermMatch] -> ResultsPrinter ()
+printList r ms = liftIO $
+    forM_ ms $ \m -> do
+        printTermAndOccurrences r
+        printAliases r
+        printFilePath m
+        printSHAs r
+        printRemovalReason r
+        putStr "\n"
+
+printHeader :: String -> IO ()
+printHeader v = do
+    setSGR [SetConsoleIntensity BoldIntensity]
+    putStr v
+    setSGR [SetConsoleIntensity NormalIntensity]
+
+printTermAndOccurrences :: TermResults -> IO ()
+printTermAndOccurrences r = do
+    setSGR [SetColor Foreground Dull (termColor r)]
+    setSGR [SetConsoleIntensity BoldIntensity]
+    putStr "  "
+    setSGR [SetUnderlining SingleUnderline]
+    putStr $ trTerm r
+    setSGR [Reset]
+
+    setSGR [SetColor Foreground Vivid Cyan]
+    setSGR [SetConsoleIntensity NormalIntensity]
+    putStr " ("
+    putStr $ pluralize (totalFileCount r) "file" "files"
+    putStr ", "
+    putStr $ pluralize (totalOccurrenceCount r) "occurrence" "occurrences"
+    putStr ")"
+    setSGR [Reset]
+    putStr "\n"
+
+printAliases :: TermResults -> IO ()
+printAliases r =
+    if null remainingAliases
```

Comment:
is there a better way to do this - basically discard one path based on a bool value?